generators/client/templates/angularjs/src/main/webapp/app/admin/user-management/_user-management-dialog.html

```diff
@@ -35,7 +35,7 @@ <h4 class="modal-title" id="myUserLabel" data-translate="userManagement.home.cre
         <div class="form-group">
             <label class="control-label" data-translate="userManagement.login">Login</label>
             <input type="text" class="form-control" name="login"
-                   ng-model="vm.user.login" ng-required="vm.user.id == null" ng-minlength=1 ng-maxlength=50 ng-pattern="/^[_'.@A-Za-z0-9-]*$/">
```

Comment:
I investigated a bit and it was like this in angular 2 as well until commit 81e27a4478a1d8cdfb0830bb252d6bb1ab3bfb66 that changed it to `required`...